src/contract_wrappers/contract_wrapper.ts

```diff
@@ -50,6 +50,12 @@ export class ContractWrapper {
         this._filterCallbacks[filterToken] = callback;
         return filterToken;
     }
+    protected unsubscribeAll(): void {
```

Comment:
lets just make this public and then remove the overrides from the subclasses

Reply:
Good catch!